db/migrations/02_add_name_to_peeps.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE peeps ADD COLUMN name VARCHAR(20);
```

Comment:
You should add this to db/migrations/01_create_chitter_table.sql (CREATE TABLE peeps(  id SERIAL PRIMARY KEY, message VARCHAR(60), name VARCHAR(20));  )